Userland/Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -2379,6 +2384,25 @@ void Document::decrement_number_of_things_delaying_the_load_event(Badge<Document
         page->client().page_did_update_resource_count(m_number_of_things_delaying_the_load_event);
 }
 
+bool Document::anything_is_delaying_the_load_event() const
+{
+    if (m_number_of_things_delaying_the_load_event > 0)
+        return true;
+
+    if (m_navigable_containers) {
+        for (auto i = 0u; i < m_navigable_containers->length(); ++i) {
```

Comment:
can we use `Document::descendant_navigables()` to get all descendant navigables instead of adding `m_navigable_containers`?